tests/core/service/logging/test_configuration.py

```diff
@@ -0,0 +1,26 @@
+import pytest
+
+from core.config import CannotLoadConfiguration
+from core.service.logging.configuration import LoggingConfiguration
+
+
+def test_cloudwatch_region_none():
+    # If cloudwatch is not enabled, no validation is needed.
+    config = LoggingConfiguration(cloudwatch=False, cloudwatch_region=None)
+    assert config.cloudwatch_region is None
+    assert config.cloudwatch is False
+
+    # If cloudwatch is enabled, region must be provided.
+    with pytest.raises(CannotLoadConfiguration):
+        LoggingConfiguration(cloudwatch=True, cloudwatch_region=None)
```

Comment:
Should we check the message to ensure that we're not just seeing a `CannotLoadConfiguration` exception, but the expected one?